### PR TITLE
Baseline Angular app for wibi

### PR DESCRIPTION
**Pull request by Borgaard**

### CODE REVIEW

File: project/static/wibi/app/router.js [hunk 1]
```diff
@@ -1,24 +1,7 @@
...
-    the controller (last step before the template) and decorates it
-
-*/
\ No newline at end of file
+angular.module('wibi-routes', []).config(['$routeProvider', '$locationProvider', function($routeProvider, $locationProvider) {
+    $routeProvider.when('/', {
+      templateUrl: '/static/templates/application.html',
+      controller: 'homeCtrl'
+    })
+    $locationProvider.html5Mode(true); //enables url change abilities without refreshing browser
```
Comment (natsn) [inline project/static/wibi/app/router.js]: will this also route the URLs if they hit them from the start?

File: project/static/wibi/bower.json [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
-    "ember-resolver": "~0.1.15",
-    "loader.js": "ember-cli/loader.js#3.2.0",
-    "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.3",
-    "ember-cli-test-loader": "ember-cli-test-loader#0.1.3",
-    "ember-load-initializers": "ember-cli/ember-load-initializers#0.0.2",
-    "ember-qunit": "0.3.0",
-    "ember-qunit-notifications": "0.0.7",
-    "qunit": "~1.17.1"
+    "qunit": "~1.17.1",
+    "angular": "~1.3.15",
```
Comment (natsn) [inline project/static/wibi/bower.json]: Do you know angular 2?

**Comment by natsn** [comment]
This looks like a great start! I'm going to merge and try it out on my system :) :+1:

File: project/static/wibi/bower.json (hunk 2 above)
Comment (Borgaard) [inline project/static/wibi/bower.json]: Nope, just Angular 1. It has a much larger population with support and from what I understand, not a lot of Angular 1 developers are moving over to Angular 2. Some of the dependencies might require substitution and have a bit of a learning curve as well.

Comment (natsn) [inline project/static/wibi/bower.json]: I've heard is very very different and they dropped a lot of things.

File: project/static/wibi/app/router.js (hunk 1 above)
Comment (Borgaard) [inline project/static/wibi/app/router.js]: What do you mean?

Comment (natsn) [inline project/static/wibi/app/router.js]: :+1:

**Pull request merged**